modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenConfig.java

```diff
@@ -66,6 +66,8 @@
 
     String outputFolder();
 
+    String testOutputFolder();
```

Comment:
Why have this and the getter?

Reply:
runtime files tree and test files tree could be in different folders.
ex. in Java there are different folders:
${project.build.directory}/generated-sources and ${project.build.directory}/generated-test-sources

currently test files get into ${project.build.directory}/generated-sources


there are 2 main issues with having test files in prod artifact:
1. test files go to production artifact
2. production code gets test dependencies

I suspect that currently majority of java developers just disable generating the test files

Reply:
I am sorry,  @spacether. It seems I didn't get your question. Both - outputFolder and testOutputFolder are getters. But `outputFolder` returns folder for a prod code, but `testOutputFolder` for tests

Reply:
What is the difference between getTestOuputFolder and testOutputDir?

Reply:
If one of them is a path can we put path in its name